cmake/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Minimum CMake required
+cmake_minimum_required(VERSION 3.5)
+
+# Project
+project(onnxir C CXX)
+
+# Set C++14 as standard for the whole project
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Do we... really need this high?

IIRC Android support for c++14 is still semi-complete so I think a safe choice is 11, unless we are already stuck with 14.

Reply:
agree, 14 is probably too aggressive. Do we use any C++14 feature?

Reply:
thank you! 11 is good.